### DISCUSSION TITLE
Oracle Mode API

### DISCUSSION THREAD

**Discussion by Muwu-Xu** [asreview/asreview#1524] [Q&A]
Hi ASreview researchers, 

is there any tutorial on how to use the API to run an Oracle mode review? I try to run the API of BaseReview to do an oracle mode but I continuously encounter many errors such as regarding 'len(label)' or something. Does it mean that I would have also given a column of label_included with some manual labels to make it run? 

Best,
Muwu

**Comment by J535D165** [comment]
Dear @Muwu-Xu, as far as I know, there is no tutorial on this at the moment. Can you share the code you use to run ASReview in Oracle mode?

(such that I can help debugging)
